extensions/azurecore/src/azureResource/providers/database/databaseTreeDataProvider.ts

```diff
@@ -18,7 +18,7 @@ import { ResourceTreeDataProviderBase } from '../resourceTreeDataProviderBase';
 export class AzureResourceDatabaseTreeDataProvider extends ResourceTreeDataProviderBase<azureResource.AzureResourceDatabase> {
 
 	private static readonly containerId = 'azure.resource.providers.database.treeDataProvider.databaseContainer';
-	private static readonly containerLabel = localize('azure.resource.providers.database.treeDataProvider.databaseContainerLabel', "SQL Databases");
+	private static readonly containerLabel = localize('azure.resource.providers.database.treeDataProvider.databaseContainerLabel', "SQL database");
```

Comment:
it seems Azure portal is using plural forms:
![Screen Shot 2020-02-21 at 11 25 47 AM](https://user-images.githubusercontent.com/13777222/75065094-f927fc00-549c-11ea-82f5-c761390148aa.png)

Reply:
These names are directly from PMs on the platform side and may not necessarily align with the portal. I'll double check with them but for now considering this by design.